app/components/language-dropdown/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'div',
+  classNames: ['dropdown', 'language-dropdown', 'inline-block', 'hidden-sm', 'hidden-xs', 'pull-right'],
```

Comment:
You've probably figured this out already in further work but the dropdown component needs to be reused on the login page so you probably don't want most of these classes here..